mris_register_to_volume/mris_register_to_label.cpp

```diff
@@ -125,7 +125,7 @@ static void argnerr(char *option, int n);
 static void dump_options(FILE *fp);
 static int  singledash(char *flag);
 #include "tags.h"
-static int istringnmatch(char *str1, char *str2, int n);
+static int istringnmatch(const char *str1, const char *str2, int n);
```

Comment:
I believe this pops up in quite a few files. Again, I didn't refactor because if `std::string` is used instead, then there's probably better alternatives.